src/store/modules/layers.store.js

```diff
@@ -4,13 +4,35 @@ import LayerTypes from '@/api/layers/LayerTypes.enum'
 import { getExtentForProjection } from '@/utils/extentUtils.js'
 import { getGpxExtent } from '@/utils/gpxUtils.js'
 import { getKmlExtent, parseKmlName } from '@/utils/kmlUtils'
+import { ActiveLayerConfig } from '@/utils/layerUtils'
 import log from '@/utils/logging'
 
 const getActiveLayerById = (state, layerId) =>
     state.activeLayers.find((layer) => layer.getID() === layerId)
 const removeActiveLayerById = (state, layerId) =>
     state.activeLayers.filter((layer) => layer.getID() !== layerId)
 
+const cloneActiveLayerConfig = (getters, layer) => {
+    const clone = getters.getLayerConfigById(layer.id)?.clone() ?? null
+    if (clone) {
+        if (layer.visible != undefined) {
+            clone.visible = layer.visible
+        }
+        if (layer.opacity != undefined) {
+            clone.opacity = layer.opacity
+        }
```

Comment:
`!=` => `!==` 😉

Reply:
No I want `!=` in order to also catch `!== null` 😉